fairseq/sequence_generator.py

```diff
@@ -421,7 +421,7 @@ def _generate(
                 break
             if self.search.stop_on_max_len and step >= max_len:
                 break
-            assert step < max_len
+            assert step <= max_len
```

Comment:
is this change related?

Reply:
No, it is not, but the tests are constantly failing in all pull requests due to this line and I actually understand the logic behind `step < max_len`. Why it can't be equal to `max_len` at that point.